Makefile

```diff
@@ -33,6 +33,7 @@ lint:
 	npm run lint --prefix ./job_runner
 	npm run lint --prefix ./lib
 install:
+	npm i
```

Comment:
I think this resolves the same thing that #34 addresses

Reply:
Narrator: it does not

Reply:
this project has a NARRATOR! wee!